src/components/game/Tutorial.vue

```diff
@@ -18,12 +18,12 @@
         <div class="texts">
           <div class="title">How to play - Keys</div>
           <div>
-            Each track has an asscoiated key, for a 4 track game, D, F, J, K,
+            Each track has an associated key, for a 4 track game, D, F, J, K,
             associates to track 1-4 respectively.
           </div>
           <img class="logo" :src="img.a" />
           <div>
-            On mobile devices, simply tap the white hit line to toggle the
+            On mobile devices, simply tap the hit line to toggle the
```

Comment:
What is a *hit line*? You probably specified it somewhere else in your UI, so probably just mark this as resolved.

Reply:
Yeah, I'll probably make a clearer tutorial image to demonstrate this in the future